app/views/components/pickadate.html

```diff
@@ -4,3 +4,8 @@
   </span>
   <input type="text" placeholder="Select a date..." class="form-control" ng-disabled="disabled">
 </div>
+<div ng-if="(minDate || maxDate) && (minDate != '' || maxDate != '')" class="help-block row" translate>
+   <label class="valid-range-msg">Valid Range:</label>
+   <span class="margin-left-10" ng-if="minDate && minDate!=''">Min Date- {{minDate | dateRangeFormat}} </span>   
```

Comment:
I think it would be nice for it to read `Valid Range: Aug 24, 2016 - Sep 23, 2016`. I know that causes a problem when only one end of the range is set.  Maybe you could only show the `Min` or `Max` text if the other end is not set. Something like `Valid Range: Aug 24, 2016 or later`. Even that doesn't sound quite right. Idk what is best.

If you end up leaving it how it is I think it needs a space before the dash (`Min Date - Sep 23, 2016`). Also, I don't think you need the `Date` text here as the string that follows it is clearly a date.

@hlbraddock Do we even need to have this text? I think the datepicker makes it pretty clear you can't select other dates but seeing it before the datepicker modal is opened might be useful:
![screen shot 2016-09-29 at 2 21 32 pm](https://cloud.githubusercontent.com/assets/756501/18972966/111e3dac-8650-11e6-8c81-51b361774c61.png)